tests-functional/tests/test_contact_request.py

```diff
@@ -0,0 +1,99 @@
+from time import sleep
+from uuid import uuid4
+import pytest
+from test_cases import OneToOneMessageTestCase
+from constants import DEFAULT_DISPLAY_NAME
+from clients.signals import SignalType
+
+@pytest.mark.rpc
+class TestContactRequests(OneToOneMessageTestCase):
+
+
+    @pytest.mark.dependency(name="test_contact_request_baseline")
+    def test_contact_request_baseline(self, contact_request_count=1):
+
+        await_signals = [
+            SignalType.MESSAGES_NEW.value,
+            SignalType.MESSAGE_DELIVERED.value,
+        ]
+
+        for i in range(contact_request_count):
+
+            message_text = f"test_contact_request_{i+1}_{uuid4()}"
+
+            sender = self.initialize_backend(await_signals=await_signals)
+            receiver  = self.initialize_backend(await_signals=await_signals)
+
+            pk_sender = sender.get_pubkey(DEFAULT_DISPLAY_NAME)
+            pk_receiver = receiver.get_pubkey(DEFAULT_DISPLAY_NAME)
+
+            existing_contacts = receiver.get_accounts()
+
+            response = sender.send_contact_request([{"id": pk_receiver, "message": message_text}])
```

Comment:
I think it's important to make functions aware of specific arguments. Tests shouldn't bother knowing the order and keys of `params`.

```python
def send_contact_request(self, contact_id: str, message: str):
    method = "wakuext_sendContactRequest"
    params = [{"id": contact_id, "message": message}]
    response = self.rpc_request(method, params)
    self.verify_is_valid_json_rpc_response(response)
    return response.json()
```